user_signup/signup.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Sign Up</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+<body>
+	<h2>Signup</h2>
+	<form method='post'>
+		<table>
```

Comment:
This, also, was just copied from the course - I think they were trying to get around having to use CSS for the layout. Thanks for the tip. I'll revamp it in the final version of the blog.